### PR TITLE
Added scenarios for scrollTo top of the window when navigating the topic tree and import a large channel

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
Scenario [started by @micahscopes](https://www.notion.so/learningequality/Gherkin-stories-for-new-scrollTo-route-behavior-d4e351e26bb0460eaf214d9f77dc078e) 

### Reviewer guidance

* Added in both `new import` and `import more` feature files.
* Reviewed the scenario for creating lessons
* Added the missing scenario for importing a large channel (still pending on having a testing channel with many nodes of small size)
* Deleted one feature file since all its scenarios are now covered elsewhere

### References
#7382
[Slack conversation](https://learningequality.slack.com/archives/CB37UM23A/p1597689176011200)

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by radinamatic** [comment]
Going to try to fit in another scenario for testing the missing resources before we merge this, and maybe the annoying CI issue will go to sleep by then :unamused:

**Comment by micahscopes** [comment]
> Going to try to fit in another scenario for testing the missing resources before we merge this, and maybe the annoying CI issue will go to sleep by then

Looks good so far! :partying_face:

**Comment by radinamatic** [comment]
@micahscopes Ready for review & merge!

File: integration_testing/features/super-admin/device/super-admin-import-new-content-channel-from-studio.feature [hunk 1]
```diff
@@ -78,6 +78,17 @@ Feature: Super admin imports content from Studio
       And I do not see the number of *resources selected* flag for the unchecked topic
       And I see the *0 resources selected* at the bottom
 
+  Scenario: Navigating a topic with many subtopics
+    Given I am navigating a <topic> topic enough subtopics to fill up the window height
```
Comment (jonboiser) [inline integration_testing/features/super-admin/device/super-admin-import-new-content-channel-from-studio.feature]:
```suggestion
    Given I am navigating a <topic> topic with enough subtopics to fill up the window height
```

File: integration_testing/features/super-admin/device/super-admin-import-new-content-channel-from-studio.feature [hunk 2]
```diff
@@ -78,6 +78,17 @@ Feature: Super admin imports content from Studio
...
+      And one <subtopic> subtopics also has enough in it items to fill up the window height
```
Comment (jonboiser) [inline integration_testing/features/super-admin/device/super-admin-import-new-content-channel-from-studio.feature]:
```suggestion
      And one <subtopic> subtopic also has enough items in it to fill up the window height
```

File: integration_testing/features/super-admin/device/super-admin-import-very-large-channel.feature [hunk 3]
```diff
@@ -0,0 +1,53 @@
+Feature: Super admin imports entire large channel
+    Super admin needs to be able to importing large channel with several thousands of resources
```
Comment (jonboiser) [inline integration_testing/features/super-admin/device/super-admin-import-very-large-channel.feature]:
```suggestion
    Super admin needs to be able to import a large channel with several thousands of resources
```

**Review by jonboiser — APPROVED** [review]
Looks good!

**Pull request merged**